system-addon/data/content/tippytop/top_sites.json

```diff
@@ -46,12 +26,7 @@
   },
   {
     "title": "ebay",
-    "urls": ["https://www.ebay.com", "https://www.ebay.co.uk/"],
-    "image_url": "ebay@2x.png"
-  },
-  {
-    "title": "ebay",
-    "url": "https://ebay.de",
+    "urls": ["https://www.ebay.com", "https://www.ebay.co.uk/", "https://ebay.de"],
     "image_url": "ebay-de@2x.png"
```

Comment:
Nit, but should the image not be `"ebay@2x.png"` here instead of `"ebay-de@2x.png"`?

Reply:
Note that you'll have to rename the image itself too

Reply:
yes. i blame @bryanbell !! lol